content/en/blog/2023/jaeger-exporter-collector-migration.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Migrating away from the Jaeger exporter in the Collector
+linkTitle: Migrating away from the Jaeger exporter in the Collector
```

Comment:
```suggestion
linkTitle: Jaeger Collector Exporter Migration
```

The `linkTitle` needs to be fairly short, not 100% happy with my suggestion, but it's the first that came to my mind